<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASLeaderboardSelector</class>
 <widget class="QWidget" name="ASLeaderboardSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>106</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASLeaderboardSelector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_Mode">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Mode">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_PlayersCount">
     <property name="text">
      <string>Leaderboard is empty</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
